docs/source/api/asns.rst

```diff
@@ -32,11 +32,20 @@ Request Structure
 -----------------
 .. table:: Request Query Parameters
 
-	+----------------+----------+-----------------------------------------------------------------------------------------------------+
-	| Parameter      | Required |                                 Description                                                         |
-	+================+==========+=====================================================================================================+
-	|   cachegroup   | no       | An integral, unique identifier for a Cache Group - only ANSs for this Cache Group will be returned. |
-	+----------------+----------+-----------------------------------------------------------------------------------------------------+
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| Parameter  | Required | Description                                                                                         |
+	+============+==========+=====================================================================================================+
+	| cachegroup | no       | An integral, unique identifier for a Cache Group - only ANSs for this Cache Group will be returned. |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| orderby    | no       | Choose the ordering of the results - must be the name of one of the fields of the objects in the    |
+	|            |          | ``response`` array                                                                                  |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| sortOrder  | no       | Changes the order of sorting. Either ascending (default) or descending ("desc")                     |
+	+------------+----------+-----------------------------------------------------------------------------------------------------+
+	| limit      | no       | Choose the number of results to return                                                              |
```

Comment:
This is the "*maximum* number of results", right? Like if I request ASNs with limit=10 but there's only one ASN, it'll return an array of length 1, not 10 where all but 1 are `null`.

Reply:
Yes

Reply:
would you mind clarifying that? basically just inserting "maximum" there would be fine.